### PR TITLE
Bring data up to date

### PR DESCRIPTION
**Pull request by eugenern**
Somewhat gigantic amount of changes here, making this unfortunately rather difficult to verify. I suppose the biggest concern (besides making sure this doesn't break the app, of course) is to check if one of the commits accidentally closes issue #1 in the original repo. I enabled issues in my forked repo, and added one there, but now realize it may possibly cause the PR to close an issue I didn't intend to close, so I apologize if that does occur.

I had to make a lot of other potentially debatable but not as scary decisions, mainly concerning the creation of new categories (since Reviewbrah has branched out some in the past year) and the inclusion of some, but not all, evaluations of products that weren't necessarily full-blown reviews (for example, I included the [Waffle House](https://www.youtube.com/watch?v=xk0lcWWCmcA) video, which Reviewbrah referred to as a review in one of his streams, and the steak encountered in his [Amtrak](https://www.youtube.com/watch?v=6tpmzgmr2Bs) video, but excluded the food mentioned in the [JetBlue](https://www.youtube.com/watch?v=An6YxxCorhk) video, since he stated there that he wasn't reviewing that food). So that kind of stuff is up for discussion, of course.

Hopefully this PR can end up being of contribution to the app! I'd be happy to add reviews in the future if necessary (with far smaller PRs), and possibly try to add features to the frontend, though I don't have much knowledge in that area right now.

### CODE REVIEW

**Comment by andyklimczak** [comment]
@eugenern this looks awesome, I'll check it out asap

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3860 @@
+{
```
Comment (andyklimczak) [inline package-lock.json]: do you mind removing this file? I just happen to be using yarn in this project instead

**Comment by andyklimczak** [comment]
This looks great! I'm cool to merge if you don't mind removing the npm .lock file first.

I'm working on cleaning up the frontend to give the new categories nice colors and stuff. (I'm realizing the frontend I wrote is super crap code wise, but it looks a lot nicer with this new data)

**Comment by andyklimczak** [comment]
Also, I'll catch us up to latest reviews too since I took a week to look at this!

**Comment by andyklimczak** [comment]
Updated the front end, once this merges everything should show nicely

File: package-lock.json (hunk 1 above)
Comment (eugenern) [reply package-lock.json]: Sure, I'm not too familiar with npm and CI, but included the file because the command line told me to.

**Comment by andyklimczak** [comment]
merged and deployed, thank you again!

I broke the build for the frontend yesterday (some test failing), I'll fix that tonight and the new data should show nicely

**Comment by andyklimczak** [comment]
fixed and deployed the frontend! https://thereportoftheweek-client.herokuapp.com/

funny that his average rating of 7.57 didn't change with the new data

**Comment by eugenern** [comment]
awesome stuff, and that's remarkable, such consistency from reviewbrah

**Pull request merged**